sycl/include/CL/sycl/kernel_bundle.hpp

```diff
@@ -287,6 +294,24 @@ class kernel_bundle : public detail::kernel_bundle_plain {
     return reinterpret_cast<device_image_iterator>(kernel_bundle_plain::end());
   }
 
+  template <backend Backend>
+  std::vector<typename backend_traits<Backend>::template return_type<
+      kernel_bundle<State>>>
+  get_native() {
+    std::vector<typename backend_traits<Backend>::template return_type<
+        kernel_bundle<State>>>
+        ReturnValue;
+    ReturnValue.reserve(std::distance(begin(), end()));
+
+    std::transform(begin(), end(), ReturnValue.begin(),
```

Comment:
Please, consider using simple for each loop to avoid including `algorithms` header which will slow down compilation of the user application.

Reply:
interesting point! +1